addon-test-support/index.ts

```diff
@@ -1,7 +1,11 @@
 export { default as a11yAudit } from './audit';
 export { default as a11yAuditIf } from './audit-if';
 export { setRunOptions, getRunOptions } from './run-options';
-export { setEnableA11yAudit, shouldForceAudit } from './should-force-audit';
+export {
+  setEnableA11yAudit,
+  shouldForceAudit,
+  calculateUpdatedHref as _calculateUpdatedHref,
```

Comment:
If we're planning on exporting this as public API why do we need to underscore it? Does it need to be public?

Reply:
It should be private. I only included it here to make it easier to import from tests (similar to the middleware reporter utils), but I can just import from the module file directly.